app/views/steps/_new.html.erb

```diff
@@ -1,5 +1,5 @@
 <div class="well">
-  <%= bootstrap_form_for(@step, url: protocol_steps_path(@protocol, format: :json), remote: true, authenticity_token: true, multipart: true, data: { role: "new-step-form", type: :json }) do |f| %>
+  <%= bootstrap_form_for(@step, url: protocol_steps_path(@protocol, format: :json), authenticity_token: true, multipart: true, data: { role: "new-step-form", type: :json }) do |f| %>
```

Comment:
less code but more trouble because I have to append additional data to the formData object before the submit is triggered